locale/action_names.rb

```diff
@@ -0,0 +1,7 @@
+# Autogenerated!
+_("Action with sub plans")
+_("Create architecture")
+_("Delete architecture")
+_("Update architecture")
+_("Import facts")
+_(" on ")
```

Comment:
Not much better here, shouldn't the `%{description} on %{host}` be shown here to give translator's context?